lib/revise_auth/routes.rb

```diff
@@ -12,6 +12,8 @@ def revise_auth
         patch "profile/email", to: "email#update"
         patch "profile/password", to: "password#update"
 
+        resources :password_resets, param: :token, only: [:new, :create, :edit, :update]
```

Comment:
I always forget about `param:`!

Reply:
Yeah, it's a pretty handy way to keep things tidy/explicit :)